ocs_ci/ocs/exceptions.py

```diff
@@ -618,3 +618,7 @@ class TerrafromFileNotFoundException(Exception):
 
 class IncorrectUIOptionRequested(Exception):
     pass
+
+
+class UnexpectedAttributeValue(Exception):
```

Comment:
It would be better to use Python `ValueError` instead of custom exception.

Reply:
Thank you, fixed